packages/webapp/pages/cores/index.tsx

```diff
@@ -98,11 +99,17 @@ export const PageCoreOptions = ({
 };
 
 const CorePageMobile = (): ReactElement => {
-  const { selectedProduct } = useBuyCoresContext();
+  const { selectedProduct, openCheckout } = useBuyCoresContext();
   const router = useRouter();
 
   useEffect(() => {
     if (selectedProduct) {
+      if (isIOSNative()) {
```

Comment:
We should probably start using the `iOSSupportsCoresPurchase` when working with payment 👀